torch/utils/hipify/constants.py

```diff
@@ -1,3 +1,4 @@
+# This file is deprecated.
```

Comment:
Not sure how actionable this is?

Reply:
There really shouldn't be anyone in the wild using this file directly.  It's just part of the old v1 soon-to-be-removed hipify machinery.  But if we remove the file, the BC-breaking CI check kicks in.  So we just put that as a comment.  Is there a better approach?  A print statement perhaps if someone imports this module?

Reply:
I've added warnings.warn to these functions and the constants.py file.